### PR TITLE
Allow HEAD Requests to app_agent and app_event.

### PR DESCRIPTION
**Pull request by damonkelley**
ping @ldko @somexpert

### CODE REVIEW

**Comment by somexpert** [comment]
How would you feel about adding a content_type check to some of those tests?

**Comment by damonkelley** [comment]
Can you be more specific?

**Comment by damonkelley** [comment]
The content type is already being checked when we compare the headers.

**Comment by somexpert** [comment]
Yeah, that's true. No need to check the same thing multiple times.

**Comment by somexpert** [comment]
:thumbsup:

**Comment by damonkelley** [comment]
@somexpert, I made some changes. I added support for HEAD requests to the `event_view`, so you might want to have another look.

**Comment by somexpert** [comment]
:thumbsup:

File: tests/test_views.py [hunk 1]
```diff
@@ -600,6 +636,43 @@ def test_delete_removes_event(self, rf):
...
+        head_response = views.app_event(head_request)
+        get_response = views.app_event(get_request)
+
+        expected_headers = get_response.serialize_headers()
+        actual_headers = head_response.serialize_headers()
+
+        assert expected_headers == actual_headers, 'The response headers do not match.'
+
+    def test_head_with_identifier(self, rf):
+        agent = factories.AgentFactory.create()
```
Comment (ldko) [inline tests/test_views.py]: Is this still supposed to use `AgentFactory`?

Comment (damonkelley) [inline tests/test_views.py]: Nope

Comment (damonkelley) [inline tests/test_views.py]: Actually, here it is supposed to be `AgentFactory`. But the equivalent test in `TestAppEvent` needed to be updated, which I have already done.

**Comment by ldko** [comment]
:+1:

**Pull request merged**